app/services/get-export-csv.js

```diff
@@ -170,3 +190,10 @@ var parseTotalSummaryTable = function (totalSummary) {
 var formatCapacityValue = function (capacity) {
   return parseFloat(capacity).toFixed(2) + '%'
 }
+
+var formatReductionDates = function (data) {
```

Comment:
There is a `services/date-formatter` helper. Could we add this logic in there / reuse some of the existing functions?